sys/include/net/ng_ipv6/hdr.h

```diff
@@ -287,15 +287,15 @@ static inline uint32_t ng_ipv6_hdr_get_fl(const ng_ipv6_hdr_t *hdr)
  * @return  The non-normalized Internet Checksum of the given IPv6 pseudo header.
  */
 static inline uint16_t ng_ipv6_hdr_inet_csum(uint16_t sum, ng_ipv6_hdr_t *hdr,
-                                             uint8_t prot_num, uint16_t len)
+        uint8_t prot_num, uint16_t len)
```

Comment:
this is not topic of this PR. Especially `ng_ipv6_hdr` needs some disentanglement (the header build functions are gnrc, the header and some related function as this one are not).